lib/linkify.js

```diff
@@ -0,0 +1,55 @@
+// github-style linkification for linkify-it
+//
+// Pass in a markdown-it parser instance and this will return a version fixed
+// up for the kind of linkification we're after.
+//
+// As far as I can determine, the basic algorithm is to linkify anything
+// starting with 'www.' and ending with a) a whitespace character, or b) a dot
+// followed by a whitespace character (in other words, include internal dots,
+// but exclude a trailing dot). There are plenty of cases where this results in
+// invalid links, but that's how they do it, so that's how we'll do it.
+//
+module.exports = function (parser) {
+  if (parser) {
+    var linkify = parser.linkify
+
+    linkify.set({fuzzyLink: false}) // turn off auto-linking normal hostnames
+    linkify.add('//', null) // turn off protocol-relative links
+
+    // linkify everything hostnamey starting with 'www.'
+    parser.linkify.add('www.', {
+      validate: function (text, pos, self) {
+        var match = text.slice(pos).match(/^\S*/)[0]
```

Comment:
Good point, I have to admit I wasn't aware of linkification of `<these> [types] (of links)` until today when you pointed to the linkify-it issue. I'll have to modify this.